web-api/openapi/schemas/document.yml

```diff
@@ -24,3 +24,11 @@ DocumentPropertyId:
   description: Id of a property of a document.
   $ref: './id.yml#/Id'
   example: "title"
+
+DocumentTag:
+  description: A tag of a document can be any non-empty, UTF-8-encoded string which doesn't contain a zero byte.
+  type: string
+  pattern: '[^\x00]+$'
```

Comment:
```suggestion
  pattern: '^[^\x00]+$'
```

Reply:
fixed. i also updated the regex in the rust code, for some reason i introduced another bug additionally to this one as well. now they are on par.